packages/eventual-send/src/track-turns.js

```diff
@@ -15,8 +15,58 @@ let hiddenPriorError;
 let hiddenCurrentTurn = 0;
 let hiddenCurrentEvent = 0;
 
+const env = globalThis?.process?.env;
```

Comment:
Please add a TODO to use environment-options.js instead, once it has been moved to its own package.